src/modules/ekf2/EKF/ekf_helper.cpp

```diff
@@ -603,6 +597,13 @@ void Ekf::updateHorizontalDeadReckoningstatus()
 		}
 	}
 
+#if defined(CONFIG_EKF2_GNSS)
+	// for fixed wing, deadreckon time starts after takeoff. Gives enough time for wind deadreckon to be activated
+	if (!_control_status.flags.in_air && !_params.gnss_ctrl && _control_status.flags.fixed_wing) {
```

Comment:
It seems wrong to rely on `_params.gnss_ctrl` here as well.

Rather than forcing/faking `_time_last_horizontal_aiding` could we instead do it for real preflight via the ZUPT or perhaps fake_pos fusion?